runtime/compiler/env/j9methodServer.cpp

```diff
@@ -1935,7 +1956,7 @@ TR_ResolvedRelocatableJ9JITaaSServerMethod::storeValidationRecordIfNecessary(TR:
          if ((*info)->_reloKind == reloKind)
             {
             if (isStatic)
-               inLocalList = (romClass == TR::Compiler->cls.romClassOf((TR_OpaqueClassBlock *) ((*info)->_clazz)));
+               inLocalList = (NULL == TR::Compiler->cls.romClassOf((TR_OpaqueClassBlock *) ((*info)->_clazz)));
```

Comment:
I was talking to Irwin about this and it's very likely this is a bug: the correct code would be to replace NULL with the romClass of the defining class. As it stands, we could add duplicate entries to this list maintained in the comp object and later on create multiple relocation records that are later on squashed. So we will use more scratch memory, but there are no functional implications.

Maybe we should make the right change here and then open a PR for a change in master branch.

Reply:
I see. Yeah, I will fix it in JITaaS code first, and then open a PR into master.